hplot/hstack.go

```diff
@@ -222,6 +252,13 @@ func (hs *HStack) hplot(c draw.Canvas, p *plot.Plot, h *H1D, yoffs []float64, hs
 		c.FillPolygon(h.FillColor, c.ClipPolygonXY(poly))
 	}
 
+	// FIX-ME (rmadar): this is plot many times but it has to be here
```

Comment:
we could add an extra argument to `func (hs *HStack) hplot(...)` to indicate the position in the stack.
either a boolean to indicate last, or just the current index in the stack (we know the length via the `hs *HStack` receiver).